client/src/services/ApiClient.tsx

```diff
@@ -0,0 +1,23 @@
+const apiUrl =
+  process.env.REACT_APP_ROUTING_API_URL || 'https://api.openrouteservice.org/';
+// TODO: get API_KEY securly
+const apiKey = process.env.REACT_APP_ROUTING_API_KEY || '';
+
+// request POIs
+const reqBody =
+  '{"request":"pois","geometry":{"bbox": [[-0.1068,51.504687],[-0.089934,51.5132]],"geojson":{"type":"Point","coordinates":[-0.106,51.505687]},"buffer":200}}';
+const endpoint = 'pois';
+
+const ApiClient = {
```

Comment:
We'll need to figure how we separate api calls that we make to external resources, vs ones that we make to our own server. Probably we can encapsulate them in separate hooks. But that can happen later.